wayshot/src/wayshot.rs

```diff
@@ -99,192 +68,65 @@ fn main() -> Result<(), Box<dyn Error>> {
         file_path = Some(utils::get_default_file_name(extension));
     }
 
-    let mut conn = Connection::connect_to_env().unwrap();
-    let (mut globals, _) = registry_queue_init::<WayshotState>(&conn).unwrap();
+    let wayshot_conn = WayshotConnection::new()?;
 
     if args.get_flag("listoutputs") {
-        let valid_outputs = output::get_all_outputs(&mut globals, &mut conn);
+        let valid_outputs = wayshot_conn.get_all_outputs();
         for output in valid_outputs {
             log::info!("{:#?}", output.name);
         }
         exit(1);
     }
 
-    let mut cursor_overlay: i32 = 0;
+    let mut cursor_overlay = false;
     if args.get_flag("cursor") {
-        cursor_overlay = 1;
+        cursor_overlay = true;
     }
 
-    let capture_area = if let Some(slurpregion) = args.get_one::<String>("slurp") {
-        match utils::parse_geometry(slurpregion) {
-            Some(region) => (wl_output::Transform::Normal, region),
-            None => {
-                log::error!("Invalid geometry specification");
-                exit(1);
-            }
+    let image_buffer = if let Some(slurpregion) = args.get_one::<String>("slurp") {
```

Comment:
It seems this might've missed my gaze during the last review. We use snake_case for variable names. Would you mind committing that?